Azure/communication-ui-library
# [Chat] Fix for "The messsage is deleted" accessibility announcement

# What

- fixed an issue with message indicator icon which has `status` role
- added should Announce prop for a message indicator used in MessageThread
- most of the code is just moved around

Steps to test:
1.
- send a few messages
- delete messages one by one
- check "The message is deleted" announcement

2.
- send a message
- delete the message
- check "The message is deleted" announcement

# Why



# How Tested

Chat sample

# Process & policy checklist



**Is this a breaking change?**
